service/src/main/java/com/codeforcommunity/processor/ProtectedSiteProcessorImpl.java

```diff
@@ -552,4 +565,15 @@ public void uploadSiteImage(
 
     // site.store();
   }
+
+  @Override
+  public void deleteSiteImage(JWTData userData, int imageId) {
+    checkImageExists(imageId);
+    SiteImagesRecord imagesRecord = db.selectFrom(SITE_IMAGES).where(SITE_IMAGES.ID.eq(imageId)).fetchOne();
+    if (!isAdmin(userData.getPrivilegeLevel())) {
```

Comment:
🔧 Could we use the `assertAdminOrSuperAdmin` method and call it before `checkImageExists`?

https://github.com/Code-4-Community/speak-for-the-trees-backend-v2/blob/2c48f6568629d1e57f1a7da8260b1cff38d19476/service/src/main/java/com/codeforcommunity/processor/AbstractProcessor.java#L12-L16